pinot-query-planner/src/main/java/org/apache/pinot/query/QueryEnvironment.java

```diff
@@ -151,8 +161,13 @@ private PlannerContext getPlannerContext(SqlNodeAndOptions sqlNodeAndOptions) {
         sqlNodeAndOptions.getOptions(), _envConfig, format);
   }
 
-  public Set<String> getResolvedTables() {
-    return _catalog.getResolvedTables();
+  /// @deprecated Use [#compile] and then [plan][CompiledQuery#planQuery(long) ] the returned query instead
```

Comment:
```suggestion
  /// @deprecated Use [#compile] and then [plan][CompiledQuery#planQuery(long)] the returned query instead
```
nit: the extra whitespace causes the closing bracket to actually be rendered